owlbot.py

```diff
@@ -467,12 +412,12 @@ def mypy(session):
 # ----------------------------------------------------------------------------
 s.replace(
     "noxfile.py",
-    r'"pytype",',
+    r'    "mypy",',
     '\g<0>\n    # "mypy_samples",  # TODO: uncomment when the checks pass',
```

Comment:
Out-of-date comment?

Reply:
Indeed, will remove it so that owlbot will not add it back.